### PR TITLE
Update Kafka utils to use images from Maven properties

### PR DESCRIPTION
**Pull request by gtroitsk**
### Summary

KafkaRegistry and KafkaVendor now using image defined in Maven properties. It helps track versions from one place.

Please check the relevant options

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Dependency update
- [x] Refactoring
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update
- [x] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [ ] Example scenarios has been updated / added
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by gtroitsk** [comment]
run tests

**Review by michalvavrik — APPROVED** [review]
LGTM

I would prefer to have review of @fedinskiy and @mjurc because this is very opinionated change, but they are out for the rest of this month. Maybe we can wait a little if @rsvoboda find a time to review. 

We can always tweak next month.

**Review by michalvavrik — CHANGES_REQUESTED** [review]
Ouch, I didn't scroll down, I still agree in principal, but I think all failures are related.

**Comment by gtroitsk** [comment]
run tests

File: quarkus-test-service-kafka/src/main/java/io/quarkus/test/services/containers/model/KafkaVendor.java [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-    private final String image;
+    private final String imageName;
     private final String defaultVersion;
     private final int port;
     private final KafkaRegistry registry;
 
-    KafkaVendor(String image, String defaultVersion, int port, KafkaRegistry registry) {
-        this.image = image;
-        this.defaultVersion = defaultVersion;
+    KafkaVendor(String image, int port, KafkaRegistry registry) {
```
Comment (rsvoboda) [inline quarkus-test-service-kafka/src/main/java/io/quarkus/test/services/containers/model/KafkaVendor.java:15]:
```suggestion
    KafkaVendor(String imagePropertyName, int port, KafkaRegistry registry) {
```

File: quarkus-test-service-kafka/src/main/java/io/quarkus/test/services/containers/model/KafkaRegistry.java [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
-    private final String image;
+    private final String imageName;
     private final String defaultVersion;
     private final String path;
     private final int port;
 
-    KafkaRegistry(String image, String defaultVersion, String path, int port) {
-        this.image = image;
-        this.defaultVersion = defaultVersion;
+    KafkaRegistry(String image, String path, int port) {
```
Comment (rsvoboda) [inline quarkus-test-service-kafka/src/main/java/io/quarkus/test/services/containers/model/KafkaRegistry.java:15]:
```suggestion
    KafkaRegistry(String imagePropertyName, String path, int port) {
```

**Review by rsvoboda — CHANGES_REQUESTED** [review]
2 small changes requested

I think this is OK-ish, the ask in QQE-41 is for test-suite.

+1 to wait for @mjurc and his opinion

**Review by michalvavrik — APPROVED** [review]

**Review by rsvoboda — APPROVED** [review]
Approving, I don't thing we need to wait for MichalJ. This is relatively internal change, people use the enum values and they didn't change.

**Pull request merged**